index.html

```diff
@@ -3447,6 +3447,19 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+
+      <section>
+        <h3>Representations of Verifiable Credentials</h3>
+
+        <p>
+Verifiable Credentials can utilize representations other than the standard one defined in this specification.
+They can do so provided that a mapping is defined between that representation and the standard one defined in this specifiction,
+so that deployments that choose to do so can map that representation to the standard one and process it accordingly.
+This mapping definition should state whether it is unidirectional or bidirectional.
```

Comment:
I think we should also define what we mean here by _unidirectional_ and _bidirectional_.

Reply:
Unidirectional and bidirectional are English words that already have defined meanings.

Reply:
It says "This mapping definition should state whether it is unidirectional or bidirectional", this means the mapping is responsible for providing sufficient clarity on this topic.

Mappings might choose to go into great detail on how they accomplished directionality, and it would be inappropriate to assume those details in this section of the spec.

I don't think we need to elaborate further.

Reply:
> It says "This mapping definition should state whether it is unidirectional or bidirectional", this means the mapping is responsible for providing sufficient clarity on this topic.

*We* are requiring that the mapping definition declare its directionality.

*We* should likewise define what we mean by this directionality — e.g., do we care about the [field ordering changes and date format disruptions that may result from CBOR-LD conversions](https://github.com/w3c/vc-data-model/pull/1100/files/b527a699109a737106078bfbe6d874ff73fa6c6e..d191dc124304da912f0bdb8764f787c4664f863f#r1180872835)? — such that the mapping authors may declare their mapping's directionality according to our requirements!

Reply:
@msporny I've incorporated suggestions to revise the title and first sentence.  Please re-review.

Reply:
Almost there, the title is still ambiguous, I suggested an alternate title that isn't as ambiguous.